src/App.js

```diff
@@ -15,6 +16,7 @@ function App() {
         <Route exact path="/" element={ <HomePage/>} />
```

Comment:
If you're using V6, exact is not required, as all routes are matched exact by default.